pxtblocks/blocklyloader.ts

```diff
@@ -12,7 +12,8 @@ namespace pxt.blocks {
         images: '#5C2D91',
         variables: '#A80000',
         text: '#996600',
-        lists: '#D83B01'
+        arrays: '#A94400',
+        advanced: '#3c3c3c'
```

Comment:
This is probably ok. But this change was made to enable targets to change the advanced category color, and those changes haven't been merged into v0. This doesn't affect anything, so I wouldn't bother with it,